Configuration/StandardSequences/python/Reconstruction_Data_cff.py

```diff
@@ -16,25 +16,6 @@
         qTest.cleaningThresholds.append(30.)
         qTest.flags.append('HFDigi')
              
-particleFlowRecHitHF.producers[0].qualityTests.append(
```

Comment:
On 9/30/15 4:24 PM, Salavat Abdullin wrote:
> In Configuration/StandardSequences/python/Reconstruction_Data_cff.py
> <https://github.com/cms-sw/cmssw/pull/11569#discussion_r40854056>:
> 
>> @@ -16,25 +16,6 @@
>>          qTest.cleaningThresholds.append(30.)
>>          qTest.flags.append('HFDigi')
>>               
>> -particleFlowRecHitHF.producers[0].qualityTests.append(
> 
> Slava, I take the liberty to include Michalis in the thread. He has
> asked me to remove only HF timing-based selection fragment and it's
> supposed to slightly change both Run 1/2 results. Salavat

Quoting Michalis:
""
Yes . This is deprecate since the HFDigiTime does it.
In 50ns it has no effect and in 25ns it has to be removed so that
it doesnt re-apply the cleaning that you removed by lowering severity levels
""

I suppose "no effect" is a statement after some threshold is applied on
the differences similar to what I observed in
https://github.com/cms-sw/cmssw/pull/11481#issuecomment-144541875



> … <#>
> On Wed, 30 Sep 2015, Slava Krutelyov wrote: In
> Configuration/StandardSequences/python/Reconstruction_Data_cff.py: > @@
> -16,25 +16,6 @@ > qTest.cleaningThresholds.append(30.) >
> qTest.flags.append('HFDigi') > >
> -particleFlowRecHitHF.producers[0].qualityTests.append( for some reason
> I thought that this change was not supposed to change anything as it was
> redundant. I see small changes in HF-related objects in both run1 and
> run2 data. @bachtis <https://github.com/bachtis> could you please
> clarify Thank you. — Reply to this email directly or view it on
> GitHub.[AEx02pi3qIEoxDrx-ixRaiH4JwpFmBQbks5o3EDGgaJpZM4GGgk3.gif]
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/11569/files#r40854056>.
>